site/layouts/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow:
+Sitemap: https://www.cvat.ai/sitemap.xml
```

Comment:
This doesn't seem right... why would `docs.cvat.ai` point at the sitemap for `www.cvat.ai`?